src/main/java/com/spotify/autoscaler/api/KCCAutoscalerController.java

```diff
@@ -0,0 +1,126 @@
+/*-
+ * -\-\-
+ * bigtable-autoscaler
+ * --
+ * Copyright (C) 2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.autoscaler.api;
+
+import static com.google.api.client.util.Preconditions.checkNotNull;
+
+import com.google.gson.Gson;
+import com.spotify.autoscaler.api.type.BigtableAutoscaler;
+import com.spotify.autoscaler.api.type.ReconcileRequest;
+import com.spotify.autoscaler.api.type.ReconcileResponse;
+import com.spotify.autoscaler.db.BigtableCluster;
+import com.spotify.autoscaler.db.BigtableClusterBuilder;
+import com.spotify.autoscaler.db.Database;
+import io.kubernetes.client.openapi.JSON;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import javax.inject.Inject;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Path("/kcc")
```

Comment:
I think should be more like `/kbb` but I would actually name it something like `/declarative`